### PR TITLE
perf(symbolication): avoid unnecessary redis calls

### LINKED ISSUE #65943: Figure out a way to do the `symbolicationDegraded` attribute fetching in a more batch friendly way.
See this trace: https://sentry.sentry.io/performance/sentry:31cf1253bfb14e3594dee7ded2b2cc74/

for large orgs, this call results in an extreme amount of latency. We should either figure out how to batch these redis calls, parallelize it, or figure out a way to disable the component on the frontend without that information in the serialized projects store.

Alternatively -- how often is this used? could we just turn off the frontend component for this and stop querying the value here?

### PR DESCRIPTION
**Pull request by anonrig**
Fixes https://github.com/getsentry/sentry/issues/65943

### CODE REVIEW

**Review by JoshFerge — COMMENTED** [review]
alternatively to passing in in the lpq_projects, you could, in `get_attrs`, get the lpq_projects, and then loop through the project_ids and set the `symbolicationDegraded` there as an attr, but just a small interface change. i don't know enough to approve,  would want to wait someone from the processing team to take a look

File: src/sentry/tasks/symbolication.py [hunk 1]
```diff
@@ -68,15 +71,17 @@ def should_demote_symbolication(project_id: int) -> bool:
...
         return True
-    else:
+    elif settings.SENTRY_ENABLE_AUTO_LOW_PRIORITY_QUEUE:
         try:
-            return (
-                settings.SENTRY_ENABLE_AUTO_LOW_PRIORITY_QUEUE
-                and realtime_metrics.is_lpq_project(project_id)
-            )
+            if lpq_projects:
+                return lpq_projects.__contains__(project_id)
```
Comment (Swatinem) [inline src/sentry/tasks/symbolication.py:77]: wouldn’t `project_id in lpq_projects` work here?

**Review by Swatinem — APPROVED** [review]

File: src/sentry/tasks/symbolication.py (hunk 1 above)
Comment (anonrig) [reply src/sentry/tasks/symbolication.py]: Yeah, that's nicer. Thanks for reviewing!

**Review by wedamija — COMMENTED** [review]
Thanks for the fix! One thing we could do to make this more robust is to make the call to `realtime_metrics.get_lpq_projects()` in `ProjectSummarySerializer.get_attrs`. This ensures that we'll always do this in an efficient way, rather than relying on the caller to know to pass in `lpq_projects`.

I'm also interested to know how many items are likely to be in the set of lpq projects. Is it likely to grow over time? One option here is to use https://redis.io/commands/smismember/, although I'm not sure if our version of Redis (and our redis client) support using this at the moment.

**Pull request merged**